### PR TITLE
/download/risc-v redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied redesign as per [mock-up](https://www.figma.com/file/WsMLgH1TXcU1mZ7YjqZRDY/u.com%2Fdownload%2Frisc-v?type=design&node-id=1-479&t=CjoQFe70UKFQNHyE-0)
- Added functionality for small screen dropdown select

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-12870.demos.haus/download/risc-v
- Check against design
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3875

### CODE REVIEW

File: templates/download/risc-v/tab-6.html
Comment (petesfrench) [inline templates/download/risc-v/tab-6.html:30]: extra closing div here

**Comment by mtruj013** [comment]
@lyubomir-popov @juanruitina whenever y'all have a minute could you take a look?

File: templates/download/risc-v/tab-7.html
Comment (petesfrench) [inline templates/download/risc-v/tab-7.html:27]: The other tabs don't have a `<hr>` between the buttons and link like appears here

**Comment by petesfrench** [comment]
image
These buttons looked a little cramped, this is right before the breakpoint changes from large to medium

File: templates/download/risc-v/tab-1.html
Comment (petesfrench) [inline templates/download/risc-v/tab-1.html:13]:
```suggestion
        loading="auto"
```
As this is visible on page load, we should let it auto load

File: templates/download/risc-v/tab-1.html
Comment (petesfrench) [inline templates/download/risc-v/tab-1.html:19]: I think you can remove `.is-fixed-width` on these, I tried removing them and couldn't see any difference

**Comment by petesfrench** [comment]
The only way to navigate the tab content, is to tab to the bottom of the list. Thus only the last item is accessible
[Screencast from 24-05-23 12:15:55.webm](https://github.com/canonical/ubuntu.com/assets/58276363/60ff64f3-2b1c-4f5a-83bf-be175c3d7445)

**Comment by lyubomir-popov** [comment]
Only one thing - the images vary in height, which makes the grey area resize a lot. Can we please make them all same height with css, and center them in the available area, both vertically and horizontally? The nset the grey area to an aspect ratio 3(width):2(height). The width should be 10% of the parent, height to match the aspect ratio.

File: templates/download/risc-v/tab-1.html
Comment (mtruj013) [reply templates/download/risc-v/tab-1.html]: I left it for 2 instances in which it was still needed to match the mock-up spacing, but removed the rest

**Comment by mtruj013** [comment]
Thanks @petesfrench @juanruitina and @lyubomir-popov, I believe this is ready for you all to take a look again. 

Few things to mention as discussed during stand-up:

- I've not been able to find a fix for the tab content focus bug. To not hold this up, I'll open a separate ticket.
- I was able to fix the logo sizing issue that I mentioned for tab 3, but please let me know if things still look wonky to you.

**Comment by juanruitina** [comment]
One minor visual comment: under SiFive Unmatched, the spacing between HRs and the "Ubuntu Server preinstalled image" and "Ubuntu Server live installer" headings is inconsistent (bigger) than under the other tabs. Other than that, this looks good to me, feel free to UX+1 whenever you fix this.

**Comment by lyubomir-popov** [comment]
I see my comment has been addressed, so +1 from me. Nice work!

File: templates/download/risc-v/tab-2.html
Comment (anthonydillon) [inline templates/download/risc-v/tab-2.html:1]:
```suggestion
<div tabindex="0" role="tabpanel" id="microchip-polarfire-soc-fpga-icicle-kit-tab" aria-labelledby="microchip-polarfire-soc-fpga-icicle-kit" aria-hidden="true">
```

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Looks good @mtruj013 few comments in the code

**Comment by mtruj013** [comment]
Thanks @anthonydillon, could you take a look again?

**Review by anthonydillon — APPROVED** [review]
LGTM thanks for the fixes

**Pull request merged**
